packages/twenty-server/src/modules/activity/standard-objects/activity.workspace-entity.ts

```diff
@@ -40,20 +51,25 @@ export class ActivityWorkspaceEntity extends BaseWorkspaceEntity {
 
   @WorkspaceField({
     standardId: ACTIVITY_STANDARD_FIELD_IDS.body,
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.RICH_TEXT,
     label: 'Body',
     description: 'Activity body',
-    icon: 'IconList',
+    icon: 'IconFilePencil',
+    defaultValue: "''",
   })
   body: string;
 
   @WorkspaceField({
     standardId: ACTIVITY_STANDARD_FIELD_IDS.type,
-    type: FieldMetadataType.TEXT,
+    type: FieldMetadataType.SELECT,
```

Comment:
We need a command to (for cloud + self-hosted)
- update metadata
- update values (we have a few example in the code base where we update enums)

Reply:
Shouldn't the command to update metadata be handled by the sync-metadata? I'll add the command to update values